api/src/config.js

```diff
@@ -2,13 +2,70 @@ const _ = require('underscore'),
       db = require('./db-pouch'),
       ddocExtraction = require('./ddoc-extraction'),
       translations = require('./translations'),
-      defaults = require('./config.default.json'),
+      defaults = require('../../config/standard/app_settings.json'),
       settingsService = require('./services/settings'),
       translationCache = {},
-      viewMapUtils = require('@shared-libs/view-map-utils');
+      viewMapUtils = require('@shared-libs/view-map-utils'),
+      contactForms =  require('medic-conf/src/fn/upload-contact-forms'),
+      collectForms =  require('medic-conf/src/fn/upload-collect-forms'),
+      appForms =  require('medic-conf/src/fn/upload-app-forms'),
+      dbnano = require('./db-nano'),
+      fs = require('fs'),
+      AdmZip = require('adm-zip'),
+      os = require('os');
 
 let settings = {};
 
+function uploadStandardForms() {
+  const tmpDir = os.tmpdir();
+  const options = { key: ['form'], limit: 1 };
+  db.medic.query('medic-client/doc_by_type', options, (err, result) => {
+    if (err) {
+      console.error(err);
+      return;
+    }
+    if (result.rows.length === 0) {
+      fs.mkdtemp(tmpDir, function(err, folder){
```

Comment:
I think this is incorrect as it will try to make a dir called `/tmp123456` rather than `/tmp/123456`. Have a look at [the documentation](https://nodejs.org/api/fs.html#fs_fs_mkdtemp_prefix_options_callback) where they have an example of the incorrect code (very similar to yours) and an example of correct code which appends a separator.

The API is really confusing for something conceptually so simple...

Reply:
Well that is a confusing one. I assumed  `/var/folders/jm/bjmcbm090n5ctj7br9qxl0440000gn/` was the temp folder and `TNNmhmx` was what mkdtemp created.  I'll adjust. 
```/var/folders/jm/bjmcbm090n5ctj7br9qxl0440000gn/T
/var/folders/jm/bjmcbm090n5ctj7br9qxl0440000gn/TNNmhmx```